pkg/controller/atlasproject/team_reconciler.go

```diff
@@ -26,6 +26,7 @@ import (
 func (r *AtlasProjectReconciler) teamReconcile(
 	team *akov2.AtlasTeam,
 	connectionSecretKey *client.ObjectKey,
+	projectID string,
 ) reconcile.Func {
```

Comment:
Remove unused extra public argument.

```suggestion
func (r *AtlasProjectReconciler) teamReconcile(
	team *akov2.AtlasTeam,
	connectionSecretKey *client.ObjectKey,
) reconcile.Func {
```